packages/fast-jss-manager-react/package.json

```diff
@@ -44,33 +52,41 @@
       "/dist/"
     ],
     "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
-    "transform": {
-      "^.+\\.tsx?$": "ts-jest"
-    }
   },
   "devDependencies": {
     "@microsoft/fast-tslint-rules": "^1.0.4",
     "@types/jest": "^22.1.2",
+    "@types/lodash-es": "^4.17.0",
     "@types/node": "^9.4.6",
     "@types/react": "^16.0.38",
+    "babel-core": "^6.26.0",
+    "babel-jest": "^22.4.3",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "hoist-non-react-statics": "^2.5.0",
     "jest": "^22.3.0",
+    "jss": "^9.8.0",
+    "jss-preset-default": "^4.3.0",
+    "lodash-es": "^4.0.0",
+    "prop-types": "^15.6.1",
     "react": "^16.0.0",
     "react-test-renderer": "^16.2.0",
     "ts-jest": "^22.0.4",
     "tslint": "^5.9.1",
     "typescript": "^2.7.1",
+    "uuid": "^3.0.0",
     "watch": "^1.0.2"
   },
   "peerDependencies": {
-    "react": "^16.0.0"
-  },
-  "dependencies": {
-    "@microsoft/fast-jss-manager": "^1.0.0",
-    "hoist-non-react-statics": "^2.3.1",
+    "hoist-non-react-statics": "^2.5.0",
+    "react": "^16.0.0",
     "jss": "^9.8.0",
     "jss-preset-default": "^4.3.0",
-    "lodash.isequal": "^4.5.0",
-    "prop-types": "^15.6.0",
-    "uuid": "^3.2.1"
+    "lodash-es": "^4.0.0",
+    "prop-types": "^15.0.0",
+    "uuid": "^3.0.0"
```

Comment:
It seems a little bit over kill by using ```uuid``` to generate the style and component ID. Will _.uniqueId('prefixIfItIsNeeded'), which is a basic counter, be enough for the purpose? Any dependency we added needs to be justified since size matters in final production. Even though for ```lodash```, it will be new for some user, since they may use underscore instead.

Reply:
I actually didn't know lodash had that function - I can definitely do that.

I've filed an issue more specific to removing that dependency: https://github.com/Microsoft/fast-dna/issues/204

Reply:
Sure.